src/main/scala/cromwell/engine/backend/local/LocalBackendCall.scala

```diff
@@ -35,4 +36,8 @@ case class LocalBackendCall(backend: LocalBackend,
 
   override def useCachedCall(avoidedTo: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] =
     backend.useCachedCall(avoidedTo.asInstanceOf[LocalBackendCall], this)
+
+  override def pollingInitialInterval: FiniteDuration = 10.seconds
+  override def pollingMaxInterval: FiniteDuration = 10.minutes
+  override def pollingMultiplier: Double = 1.1D
```

Comment:
Isn't it true that none of this can really apply to a `LocalBackendCall` since there's no polling?